test/ocp_qp/test_qpsolvers.cpp

```diff
@@ -145,7 +159,7 @@ TEST_CASE("mass spring example", "[QP solvers]")
 
     int ngN = 0;  // number of general constraints at last stage
 
-    double N2_values[] = {15, 5, 3};  // horizon of partially condensed QP for sparse solvers
```

Comment:
Why? Any issue with N2=3?

Reply:
Oh right, forgot about that. Not exactly an issue, more of an inconvenience. OSQP is not accurate enough, for N2=3 the tolerances for primal and dual residuals should be quite smaller to get an accurate solution. I'll add these tolerances to OSQP opts so that they can be changed from outside...

Reply:
Nvm, I just realized polishing (for high accuracy) has been turned of in OSQP. With polishing turned on, the accuracy is on par with other solvers :)

Reply:
nice :)